### ISSUE TITLE
Deployment looking for order api credentials even though deployment assets are manually downloaded

### ISSUE THREAD

**Issue by nbattala** [sassoftware/viya4-deployment#380]
Hi, 
I downloaded Viya order and defined the 'V4_CFG_DEPLOYMENT_ASSETS' and 'V4_CFG_LICENSE' properties. However, the install fails with the following error. Could you please let me know if there is something wrong with my configuration or this is a bug? I have the certificate to pass to ansible as a property so that it does not have to use viya4-orders-cli, but I do not see any such property name in the documentation. Thanks!

```bash

TASK [vdm : assets - Get Certificates] *****************************************
fatal: [localhost]: FAILED! => changed=true 
  cmd:
  - /tmp/ansible.003ipsmn/viya4-orders-cli
  - certificates
  - --file-path
  - /data/sasfd-aks/license
  - --file-name
  - certs
  delta: '0:00:01.425318'
  end: '2023-01-26 17:20:03.653024'
  msg: non-zero return code
  rc: 1
  start: '2023-01-26 17:20:02.227706'
  stderr: |-
    2023/01/26 17:20:02 INFO: no config file found
    2023/01/26 17:20:03 ERROR: Bearer token request failed: oauth2: cannot fetch token: 401 Unauthorized
    Response: {"ErrorCode" : "invalid_client", "Error" :"ClientId is Invalid"}
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>
```

**Comment by jarpat** [comment]
Hey @nbattala 

At the moment, we currently don't provide a variable to set user provided certificates, something like `V4_CFG_CERTS`. We are also looking into implementing a change where the viya4-orders-cli download is skipped if all three variables, `V4_CFG_DEPLOYMENT_ASSETS`, `V4_CFG_LICENSE`, & `V4_CFG_CERTS` are defined.

This is on our radar and actually in the current development sprint. I'll update this issue soon with a PR so you can track this.

**Comment by jarpat** [comment]
https://github.com/sassoftware/viya4-deployment/pull/383

**Comment by jarpat** [comment]
Closing, implemented in the 6.2.0 release 
https://github.com/sassoftware/viya4-deployment/releases/tag/6.2.0
